components/identities/actions/identityActions.tsx

```diff
@@ -56,6 +61,7 @@ const IdentityActions: FunctionComponent<IdentityActionsProps> = ({
   const [isAutoRenewalEnabled, setIsAutoRenewalEnabled] =
     useState<boolean>(false);
   const [allowance, setAllowance] = useState<string>("0");
+  const [hasRev, setHasRev] = useState<boolean>(false);
```

Comment:
Don't understand the naming here

Reply:
Fixed by https://github.com/starknet-id/app.starknet.id/pull/580/commits/7d46567461f754ecd7b0323ef2755fdd400e5f67